src/burnchains/bitcoin/address.rs

```diff
@@ -269,19 +632,81 @@ impl Address for BitcoinAddress {
     }
 }
 
-impl std::fmt::Display for BitcoinAddress {
+impl Address for SegwitBitcoinAddress {
+    fn to_bytes(&self) -> Vec<u8> {
+        self.bytes()
+    }
+
+    fn from_string(s: &str) -> Option<SegwitBitcoinAddress> {
+        SegwitBitcoinAddress::from_bech32(s)
+    }
+
+    fn is_burn(&self) -> bool {
```

Comment:
N.B. Make sure that PoX addresses in block-commits are all legacy Bitcoin addresses